docs/html/user_guide.rst

```diff
@@ -791,3 +791,25 @@ of ability. Some examples that you could consider include:
 
 * ``distlib`` - Packaging and distribution utilities (including functions for
   interacting with PyPI).
+
+
+.. _`Disable Keyring`:
+
+Disable Keyring
+***************
+
+Keyring in certain cases can prevent the installing of certain packages and modules
```

Comment:
Also I don't think `keyring` is mentioned anywhere else in the user guide, maybe before talking about how to turn it off, you could mention that pip doesn't come with keyring but it may try to use it if the user has it installed.